lightly/active_learning/agents/agent.py

```diff
@@ -110,7 +111,7 @@ def query(self, sampler_config: SamplerConfig, al_scorer: Scorer = None) -> List
                 An instance of a class inheriting from Scorer, e.g. a ClassificationScorer.
 
         Returns:
-            The filenames of the samples in the new labeled_set.
+            The filenames of the samples in the new labeled_set and the added filenames.
```

Comment:
This description is too minimal, please elaborate. Remember, this is not internal but what's displayed in the documentation so ideally, a user would read this and understand immediately what the returned lists are.